editoast/schemas/src/rolling_stock.rs

```diff
@@ -120,6 +120,32 @@ pub struct RollingStock {
     pub other_categories: TrainMainCategories,
 }
 
+impl RollingStock {
+    pub fn supported_signaling_systems_list(&self) -> Vec<String> {
```

Comment:
I believe the signature of the function is enough.

```suggestion
    pub fn supported_signaling_systems(&self) -> Vec<String> {
```